From f94e05110f5d670c71624c18cb479083c3ae3e51 Mon Sep 17 00:00:00 2001
From: Yaakov Selkowitz <yselkowi@redhat.com>
Date: Mon, 8 Jul 2024 11:28:30 -0400
Subject: [PATCH] Use rpm-sequoia on RHEL 10+
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

RHEL 10 also has a sufficiently new RPM with rpm-sequoia enabled.

Signed-off-by: Petr Písař <ppisar@redhat.com>
---
 librepo.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/librepo.spec b/librepo.spec
index 25dc741..62035dc 100644
--- a/librepo.spec
+++ b/librepo.spec
@@ -8,7 +8,7 @@
 %bcond_without zchunk
 %endif
 
-%if 0%{?fedora} >= 39
+%if 0%{?fedora} >= 39 || 0%{?rhel} >= 10
 %bcond_with use_gpgme
 %bcond_with use_selinux
 %else
-- 
2.45.2

